### DISCUSSION TITLE
Can the table of content have page number reference?

### DISCUSSION THREAD

**Discussion by syang** [honkit/honkit#348] [Q&A]
I am trying to use honkit to publish a printed book. For a printed book, the TOC contains the starting page number reference for each chapter.

But honkit gives the following Table of Content (not page number of the starting page)

Chapter 1: My lovely first chapter .............. 1.1
Chapter 2: My lovely 2nd chapter .............. 1.2
...
Chapter 5: My lovely 5th chapter ............... 1.5

Is there a way to make page number shown for each chapter?

**Comment by azu** [comment]
HonKit uses [ebook-convert](https://manual.calibre-ebook.com/generated/en/ebook-convert.html) for convert to PDF.
However, It looks like that ebook-convert does not have this option.

**Reply by syang** [reply]
Thanks @azu  for the reply. Do you know any alternative / tricks to achieve it? (I am trying to publish a printable book)

Also, another question regarding he printable book:

(my SUMMARY.md file)

```
# Summary

- [Preface: to my readers](README.md)
- [Chapter 1: blah 1](content/1.md)}
- [Chapter 2: blah 2](content/2.md)
- [Chapter 3: blah 3](content/3.md)
```

With this summary file structure, I found that the header of each page shows the line as "Preface: to my readers". Is there away either show the header line as the specific chapter name or show nothing?

Thanks,
